site/content/en/docs/api_sdk/sdk/highlevel-api.md

```diff
@@ -8,25 +8,29 @@ description: ''
 ## Overview
 
 This layer provides high-level APIs, allowing easier access to server operations.
-API includes _Repositories_ and _Entities_. Repositories provide management
-operations for Entitites. Entitites represent separate objects on the server
-(e.g. tasks, jobs etc).
+API includes _Repositories_ and _Entities_. _Repositories_ provide management
+operations for _Entities_. _Entities_ represent objects on the server
```

Comment:
I would suggest only using italics for the defining occurrences of the terms (once for each term). Otherwise it's going to be annoying to consistently italicize these terms everywhere - in fact, I already see them appear without italics at the bottom of the document. I don't think italics everywhere would help with readability either.

```suggestion
API includes Repositories and Entities. _Repositories_ provide management
operations for Entities. _Entities_ represent objects on the server
```

Reply:
Agree. I decided to leave them in 2 places, where the terms can be declared for a reader.